data/json/monsters/insect_spider.json

```
@@ -518,6 +518,30 @@
     ],
     "extend": { "flags": [ "DESTROYS", "PUSH_MON", "PUSH_VEH" ] }
   },
+  {
+    "id": "mon_firefly",
+    "type": "MONSTER",
+    "name": { "str": "firefly", "str_pl": "fireflies" },
+    "description": "A mutated firefly grown to the size of a dog, brightly glowing abdomen bobbing up and down as it tries to impress its unseen mate.  It looks strangely uninterested in you and everything else.",
```

```suggestion
    "description": "A mutated firefly grown to the size of a large housecat, brightly glowing abdomen bobbing up and down as it tries to impress its unseen mate.  It looks strangely uninterested in you and everything else.",
```
When I hear dog, I think a 15 kg hound more than something the size of a milk jug. Housecats are more consistent in size.